src/app/services/mark-inactive-driver-service/mark-inactive-driver.service.ts

```diff
@@ -0,0 +1,48 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpHeaders} from '@angular/common/http';
+import { Observable } from 'rxjs';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MarkInactiveDriverService {
+
+  // URL
+  private url = "http://localhost:8080/users/"
+  // private body: string;
+  // // Http options
+  //   private httpOptions = {
+  //     headers: new HttpHeaders({"Content-Type": "application/json"}),
+  //     observe: "response" as "body"
+  //   };
+
+
+
+
+  constructor(private httpClient: HttpClient) { }
+
+
+getDriverById(id: number): Observable <any>{
```

Comment:
Are the operations found in this file operations done on the user model? Marking a driver inactive is updating a user, correct? Why not include this functionality in the User service?

Reply:
@goscha01 can we integrate this into the user service?